libs/ledger-live-common/src/families/hedera/api/mirror.ts

```diff
@@ -122,7 +121,8 @@ export async function getOperationsForAccount(
     recipients.reverse();
     senders.reverse();
 
-    const hash = base64ToUrlSafeBase64(raw.transaction_hash);
+    const buffer = Buffer.from(raw.transaction_hash, "base64");
+    const hash = buffer.toString("hex");
```

Comment:
Just to note, this will not update existing transactions in the cache.
Which means that anything that this fixes, will require the user to manually use the "clear cache" setting, for it to apply to all previous transactions.